brightcove-exoplayer/BasicSsaiSampleApp/src/main/java/com/brightcove/player/samples/ssai/basic/MainActivity.java

```diff
@@ -0,0 +1,75 @@
+package com.brightcove.player.samples.ssai.basic;
+
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.brightcove.player.event.Event;
+import com.brightcove.player.event.EventEmitter;
+import com.brightcove.player.event.EventListener;
+import com.brightcove.player.event.EventType;
+import com.brightcove.player.view.BrightcoveExoPlayerVideoView;
+import com.brightcove.player.view.BrightcovePlayer;
+import com.brightcove.ssai.SSAIComponent;
+
+public class MainActivity extends BrightcovePlayer {
+    // Private class constants
+    private final String TAG = this.getClass().getSimpleName();
+
+    // Private instance variables
+
+    // The SSAI plugin VMAP data URL, which tells the plugin when to
+    // send tracking beacons, when to hide the player controls and
+    // what the click through URL for the ads shoud be.  The VMAP data
+    // will also identify what the companion ad should be and what
+    // it's click through URL is.
+    private String ssaiAdDataUrl = "http://once.unicornmedia.com/now/ads/vmap/od/auto/c501c3ee-7f1c-4020-aa6d-0b1ef0bbd4a9/202ef8bb-0d9d-4f6f-bd18-f45aa3010fe6/8a146f45-9fac-462e-a111-de60ec96198b/content.once";
+
+    private SSAIComponent plugin;
+
+    public SSAIComponent getSsaiUxPlugin() {
+        return plugin;
+    }
```

Comment:
This method isn't used anywhere. Its OnceUx counterpart wasn't used anywhere either.